tests/unit/SpecialResolutionSummary.spec.ts

```diff
@@ -114,25 +113,29 @@ describe('Special Resolution Review', () => {
   })
 
   it('rules', async () => {
-    await store.setSpecialResolutionRules({
```

Comment:
Store methods are not async, so awaiting them does nothing.

However, in some cases we need a small delay after these to let other code (watchers, computed) to update themselves.